src/Soil/energy_hydrology.jl

```diff
@@ -239,7 +242,7 @@ function ClimaLSM.make_rhs(model::EnergyHydrology{FT}) where {FT}
         # This has to come last
         dss!(dY, model.domain)
```

Comment:
with CTS, was this removed from the tendency function?

Reply:
We can remove it from the tendency if we include it in the `CTS.ClimaODEFunction` when we construct the `ODEProblem` (like [this](https://github.com/CliMA/ClimaLSM.jl/blob/js/implicit-solver/src/Soil/examples/imex_testing_modified_picard.jl#L131-L140)). We're not currently doing that in main though, only on the implicit solver branch.

Reply:
Does this mean we either define our tendency function with it (and use ODE's infrastructure completely) or else we remove it from the tendency and pass it via  CTS.ClimaODEFunction() but then are required to use ClimaTimesteppers? or can we use e.g. a CTS.ClimaODEFunction but use an ODE timestepper?

Reply:
(I suspect the latter, but just want to make sure!)

Reply:
Unfortunately, we can't use a CTS problem setup (e.g. `CTS.ClimaODEFunction`) with an ODE timestepping algorithm. 

If we wanted to use CTS to setup the `ODEProblem` but then an ODE algorithm to step it, we could add the dss step to the end of both the explicit and implicit tendencies, and use a `SplitODEProblem` (instead of an `ODEProblem`) with both an implicit and an explicit tendency (no dss field). However, this approach became numerically unstable after 300 days when used in ClimaAtmos, so it's probably a better approach to remove dss from the tendencies and include it in the problem instead, even though this won't be compatible with ODE algorithms.